### PR TITLE
Issue https://github.com/h6ah4i/android-advancedrecyclerview/issues/1…

### PR DESCRIPTION
**Pull request by thefex**
…42 fix

### CODE REVIEW

File: library/src/main/java/com/h6ah4i/android/widget/advrecyclerview/utils/BaseWrapperAdapter.java [hunk 1]
```diff
@@ -121,7 +133,7 @@ protected void onHandleWrappedAdapterItemRangeChanged(int positionStart, int ite
     }
 
     protected void onHandleWrappedAdapterItemRangeChanged(int positionStart, int itemCount, Object payload) {
-        notifyItemRangeChanged(positionStart, itemCount, payload);
```
Comment (h6ah4i) [inline library/src/main/java/com/h6ah4i/android/widget/advrecyclerview/utils/BaseWrapperAdapter.java]: @thefex Why did you modified this line?

**Comment by thefex** [comment]
Reverted change in this method, my fault - sorry I have written fix for old version for library (which didn't had notifyItemRangeChanged signature with Object) - then I updated source and forgot to fix that.

**Comment by h6ah4i** [comment]
Thank you for fixing the `notifyItemRangeChanged()` method. It looks good to me ; )

I am going to merge it manually, because I want to merge it into the `develop` branch.

**Comment by h6ah4i** [comment]
Done. It will be included in v0.8.4. Thanks!

**Pull request closed without merging**